### DISCUSSION TITLE
Angularエンジニアの採用、うまくいってますか？

### DISCUSSION THREAD

**Discussion by yamagenii** [StartupAngular/discussions#16] [General]

**Comment by scrpgil** [comment]
スレッド1

**Reply by scrpgil** [reply]
# Angularの採用周り
## 質問
- 苦戦をしている。人がいないのが正直になっている。
- 採用要件にAngularが入っていなくて、pythonとかの人が、入ってきて後で習得してもらっているが、少ない
- 苦戦している。React，Vueに比べると掲載数も少ない
- 同じく苦戦している。

## 候補者のレベル感
- 新卒
 - 元々、個人開発でAngularを使っていてインターンで入社したケースもある。
- 中途
 - React、VueのSPA開発がある人でフィルターをかけている。
 -基本は中途採用の会社が多い。

## 結論
- Angular専任だとすごく候補者が少なくなってしまう。
- 別の技術スタックを持ってる方にキャッチアップしてもらうほうが、採用としては早いし、実例もある。
